src/components/Menu/Menu.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+export default function Menu({ changePage }) {
+  return (
+    <div className='menu'>
```

Comment:
Consider refactoring to generate each ring in a more 'react' way. Mapping over a helper and passing in 'favorites, planets' etc to create each ring